src/definitions.ts

```diff
@@ -521,8 +512,10 @@ export interface AdMobRewardItem {
 }
 
 export interface AdMobError {
-  reason: string;
   code: number;
+  message: string;
+  cause: string;
+  domain: string;
```

Comment:
comment explaining what does any of this mean would be nice

```
/** My meaning is...**/
domain: string;
```

Reply:
Good idea. I added explain comment.